lib/app.js

```diff
@@ -247,6 +247,17 @@ function bindMessenger(Application, agent) {
     constructor(options) {
       super(options);
 
+      // enable app to send to a random agent
+      this.messenger.sendRandom = ((action, data) => {
```

Comment:
这里不对吧，sendRandom 特指的是 agent -> app，而不是反过来

Reply:
egg 项目的 egg/lib/core/messenger/ipc.js 和 local.js 的 sendRandom() 注释写的是如果在 app 调用该方法，会直接发送给 agent。所以在这里加了这么一段，让 app 的 sendRandom() 也能生效
``` js
  /**
   * send message to one app worker by random
   * - if it's running in agent, it will send to one of app workers
   * - if it's running in app, it will send to agent
   * @param {String} action - message key
   * @param {Object} data - message value
   * @return {Messenger} this
   */
  sendRandom(action, data) { ... }
```